hlapipeline/alignimages.py

```diff
@@ -243,7 +266,7 @@ def perform_align(input_list, archive=False, clobber=False, update_hdr_wcs=False
         if total_num_sources < MIN_OBSERVABLE_THRESHOLD:
             print("Not enough sources ({}) found in image {}".format(total_num_sources,imgname))
             filteredTable[index]['status'] = 1
-            return(filteredTable)
+            return(filteredTable) #TODO: Should this update the status of all images in filteredTable with the value 1? What about processMsg?
```

Comment:
Yes,  I was going to fix this problem.  It should be filteredTable['status'][:] = 1.   We could update the processMsg which was originally designed for the 'doProcess=False' explanation.  The total number of sources is stored about in the filtered table, so we do have some information.  Of course this is only until we hit this rejection criterion.  You should feel free to update the filteredTable status as above.

Reply:
ok cool. I'll fix that, and I think there are a couple of instances of this sort of thing elsewhere in the code. I'll update accordingly.